src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -118,16 +119,17 @@ function ReportActionItemMessageEdit(props) {
         });
 
         return () => {
-            // Skip if this is not the focused message so the other edit composer stays focused
-            if (!isFocusedRef.current) {
+            // Skip if this is not the focused message so the other edit composer stays focused.
```

Comment:
I'm sort of confused by this comment. Do we mean something like this?

```suggestion
            // Skip showing the main composer if the context menu is active for this reportAction. We want the "Edit composer" text input stay focused in this case.
```